### PR TITLE
Bump up the order to highlight the importance of sample asset

### PR DESCRIPTION
**Pull request by amzn-rhhong**
We have too many GitHub issue created that doesn't have asset attached to. This change is designed to highlight the important to attach asset to a reported issue, and hopefully increase the chances that customer actually provided the asset at the beginning.
Signed-off-by: rhhong <rhhong@amazon.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ labels: 'needs-triage,needs-sig,kind/bug'
 **Describe the bug**
 A clear and concise description of what the bug is. Try to isolate the issue to help the community to reproduce it easily and increase chances for a fast fix.
 
+**Assets required**
+[!Important!] Please provide sample assets needed to reproduce the issue as much as possible.
```
Comment (lmbr-pip) [inline .github/ISSUE_TEMPLATE/bug_template.md:13]:
```suggestion
[!Important!] Please provide sample assets needed to reproduce the issue, either as an [attachment](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files) or as a link to a public asset in Github.

```

**Review by lmbr-pip — COMMENTED** [review]
Consider adding information about how folks can attach assets (some will not be familar with GitHub). Users should be encouraged to also provide links to assets in GitHub IMHO but we may need to call out license restrictions. 

Feel free to ignore or edit suggestion.

**Review by micronAMZN — APPROVED** [review]
Approve pending @lmbr-pip's suggestion.

**Review by hultonha — APPROVED** [review]
Looks good and agree with @lmbr-pip's suggestion :+1: Thanks for the fast turnaround @amzn-rhhong!

**Review by amzn-jillich — APPROVED** [review]

**Comment by hultonha** [comment]
@amzn-rhhong You should be able to ask @tjmichaels for an exception with this change so you don't need to run AR for a text-only change

**Pull request merged**